scripts/test/bs-kill.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+/**
+ * This script stops all BrowserStack workers that are currently running.
+ * Useful when troubleshooting BrowserStack issues locally.
+ */
+const { printLog, runMain, timeout } = require('../lib/execution-utils')
+const { browserStackRequest } = require('../lib/bs-utils')
+
+const MINIMUM_WORKER_LIFE = 30_000
+
+const BS_WORKERS_URL = 'https://api.browserstack.com/5/workers'
+const BS_WORKER_URL = (workerId) => `https://api.browserstack.com/5/worker/${workerId}`
```

Comment:
**💬 suggestion:** ‏it's quite unusual to have uppercased functions, what about sticking to regular casing / function declaration?